pkg/libvmi/config.go

```diff
@@ -109,6 +129,14 @@ func newSecretVolume(secretName, volumeName, label string) v1.Volume {
 	}
 }
 
+func newSysprepVolume(volumeName string, source *v1.SysprepSource) v1.Volume {
+	return v1.Volume{Name: volumeName,
```

Comment:
@aliamerj Could you please make `make lint` happy?
I think you should move `Name: volumeName,` in a new line.